tools/shell/linenoise.cpp

```diff
@@ -953,11 +1037,15 @@ static int linenoiseEdit(
             break;
         case CTRL_T: /* ctrl-t, swaps current character with previous. */
             if (l.pos > 0 && l.pos < l.len) {
-                int aux = buf[l.pos - 1];
-                buf[l.pos - 1] = buf[l.pos];
-                buf[l.pos] = aux;
-                if (l.pos != l.len - 1)
-                    l.pos++;
+                char tempBuffer[128];
```

Comment:
we always use `tmp` instead of `temp` for variable names

Reply:
renamed it